CMakeLists.txt

```diff
@@ -684,6 +688,24 @@ ELSE(QMC_CUDA)
   MESSAGE(STATUS "Disabling CUDA")
 ENDIF(QMC_CUDA)
 
+SET(XRAY_PROFILE FALSE CACHE BOOL "Use llvm xray profiling")
```

Comment:
Please add comments.

Reply:
I left help strings and documentation in examples/profiling. Could add documentation to the manual or wiki but I don't think this is necessary for this to be merged..

Reply:
I didn't mean that. Wiki or manual is not necessary for this at the moment. I was asking for just one or two lines of comment what this is. Since it has been moved to Clang, the cached variable is self-explained.